README.md

```diff
@@ -12,12 +12,12 @@ SkyPilot is a framework for easily running machine learning workloads[^1] on any
 
 Use the clouds **easily** and **cost effectively**, without needing cloud infra expertise.
 
-_Ease of use & productivity_
+_Ease of use_
 * **Run existing projects on the cloud** with zero code changes
-* **Easily manage jobs** across multiple clusters
-* **Automatic fail-over** to find scarce resources (GPUs) across regions and clouds
-* **Store datasets on the cloud** and access them like you would on a local file system 
-* **No cloud lock-in** – seamlessly run your code across different cloud providers (AWS, Azure or GCP)
+* Use a **unified interface** to run on any cloud, without vendor lock-in (currently AWS, Azure, GCP)
```

Comment:
nit: remove `Use a ` to start directly with the main keyword. e.g. `**Unified interface** to run jobs on any cloud, without vendor lock-in (currently AWS, Azure, GCP)`

Reply:
Debated offline; keeping the current.